File: clients/typescript-angular/API/Client/Trade.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Trade {

        timestamp?: date;

        symbol?: string;

        side?: string;

        size?: number;

        price?: number;

        tickDirection?: string;

        trdMatchID?: string;

        grossValue?: number;

        homeNotional?: number;

        foreignNotional?: number;

        id?: number;
    }

}

File: clients/async-scala/src/main/scala/io/swagger/client/model/Trade.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Trade (
  timestamp: Date,
symbol: String,
side: Option[String],
size: Option[Number],
price: Option[Double],
tickDirection: Option[String],
trdMatchID: Option[String],
grossValue: Option[Number],
homeNotional: Option[Double],
foreignNotional: Option[Double]
)
